--- a/network.c
+++ b/network.c
@@ -160,7 +160,7 @@ loopback_setup (void)
     die_with_error ("loopback: Failed to bind NETLINK_ROUTE socket");
 
   header = rtnl_setup_request (buffer, RTM_NEWADDR,
-                               NLM_F_CREATE | NLM_F_EXCL | NLM_F_ACK,
+                               NLM_F_CREATE | NLM_F_ACK,
                                sizeof (struct ifaddrmsg));
   addmsg = NLMSG_DATA (header);
 
